docs/spec.md

```diff
@@ -3942,27 +3942,28 @@ separate outputs to improve clarity
 
 | Name      | Type                                                    | Constraints |
 |-----------|---------------------------------------------------------|-------------|
-| `results` | variadic number of tensors, quantized tensors or tokens | (C2), (C3)  |
+| `results` | variadic number of tensors, quantized tensors or tokens | (C2-C4)     |
 
 #### Constraints
 
 * (C1) `channel_type` is defined as:
```

Comment:
Is there any plan to back this value with an enum? It looks like we don't document what value should be used if you wanted `HOST_TO_DEVICE` for example. An int64_t enum in C++ should help resolve that issue for StableHLO C++ users.

Additionally, we could make the pretty printer/parser for channel handle attr accept the string values for valid specced enums.

Reply:
There is no such plan at the moment, but there a plan to remove this value altogether: https://github.com/openxla/stablehlo/issues/666 :)